src/applications/ezr/containers/App.jsx

```diff
@@ -77,16 +97,16 @@ const App = props => {
   // Add Datadog UX monitoring to the application
   useBrowserMonitoring();
 
-  return isAppLoading || !isProdEnabled ? (
-    <va-loading-indicator
-      message={content['load-app']}
-      class="vads-u-margin-y--4"
-      set-focus
-    />
+  return isAuthOnlyEnabled &&
+    !window.location.href.startsWith('http://localhost') ? (
```

Comment:
There is an environment util called `isLocalhost` that handles this. It would be much cleaner to review that method and use it here instead of manually checking the window location.